### PR TITLE
EMBCESSMOD-4900:  Pets and Household Members tabs merge

### PR DESCRIPTION
**Pull request by TimaQT**
EMBCESSMOD-4900 
The Pets and Household Members tabs consolidated into a single step within the ESS Wizard.

### CODE REVIEW

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/ess-file-review/ess-file-review.component.html [hunk 1]
```diff
@@ -92,7 +92,7 @@
       <div class="row">
         <div class="col-12">
           <div class="review-panel data-panel">
-            <label class="tab-subtitle">Household Members</label>
+            <label class="tab-subtitle">Household Members & Pets</label>
```
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/ess-file-review/ess-file-review.component.html:95]: I recommend to keep the review page unchanged - only the edit pets should send the user to the household member and pets page

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/animals/animals.component.ts [hunk 2]
```diff
@@ -64,6 +65,8 @@ export class AnimalsComponent implements OnInit, OnDestroy {
       this.animalsForm.get('hasPetsFood').updateValueAndValidity();
```
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/animals/animals.component.ts:65]: aren't we removing pet food question?

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/household-members/household-members.component.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <div class="heading-container">
     <div class="row">
       <div class="col-md-12">
-        <span class="wizard-tab-heading">Household Members</span>
+        <span class="wizard-tab-heading">Household Members & Pets</span>
```
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/household-members/household-members.component.html:5]: shouldn't this be moved to the new composite component?

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/ess-file-review/ess-file-review.component.html (hunk 1 above)
Comment (TimaQT) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/ess-file-review/ess-file-review.component.html]: I returned the previous state ( except the changes we have to adjust based on 2 questions being removed). And there is no link to the step from the review file

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/animals/animals.component.ts (hunk 2 above)
Comment (TimaQT) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/ess-file-components/animals/animals.component.ts]: Done

**Review by ytqsl — APPROVED** [review]
I believe the pet care plans and food should be removed from the backend too

**Pull request merged**